onboarding/source/amb_light_service.c

```diff
@@ -33,21 +47,62 @@ static void lightServiceTask(void * pvParameters);
 
 obc_error_code_t initLightService(void) {
     /* USER CODE BEGIN */
-    // Create the task and queue here. Return error code if task/queue was not created successfully.
+    BaseType_t xReturned = pdFAIL;
+        if (lightServiceHandle == NULL) {
+            xReturned = xTaskCreate(lightServiceTask,
```

Comment:
We use the statically-allocated version of tasks, queues, etc. In this case, use `xTaskCreateStatic` instead.